references-in-rfcxml.md

```diff
@@ -11,7 +11,7 @@ dateCreated: 2021-11-04T23:45:18.949Z
 # Introduction
 RFCXML uses references that are encoded in a format called BibXML, as defined in RFC 7991. To simplify the process of inserting references, the IETF maintains a set of online citation libraries as follows:
 * For RFCs the official site is [rfc-editor.org](https://rfc-editor.org/). The information page for each RFC has a link to the BibXML file.
-* For RFCs, Internet-Drafts, and documents produced by the W3C, 3GPP, IANA and NIST the current site is [xml2rfc.tools.ietf.org]().  Many of these sets of references can be downloaded as a set for offline access.
+* For RFCs, Internet-Drafts, and documents produced by the W3C, 3GPP, IANA and NIST, the new site is [bib.ietf.org](https://bib.ietf.org) (at the time of writing, this site has only just gone live and some usability issues are still to be resolved). Many of these sets of references can be downloaded as a set for offline access.
```

Comment:
A big usability issue is that the site doesn't actually provide the URL that an author needs to create an xi:include, but the link to bib.ietf.org does need to be updated on this page. An issue needs to be opened for this page to track these usability issues and then remove this line when we think bib.ietf.org is really ready to go.

Reply:
> A big usability issue is that the site doesn't actually provide the URL that an author needs to create an xi:include, 

Yes indeed - multiple eyeroll emojis.  I will add an issue.